byway/pom.xml

```diff
@@ -24,6 +24,16 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>com.google.code.gson</groupId>
+      <artifactId>gson</artifactId>
+      <version>2.8.6</version>
+    </dependency>
```

Comment:
You have the gson dependency listed twice now.

Reply:
Good catch, thanks for pointing that out.